source/leuven/wice_advanced_guide.rst

```diff
@@ -9,15 +9,68 @@ wICE advanced guide
 Compiling software
 ------------------
 
-The wICE compute nodes feature Intel processors nicknamed IceLake, which are
-the successors of the SkyLake and CascadeLake architectures which you can find
-on Genius. Although architectural differences are rather small, it is highly
-recommended to compile a version of your code specifically for wICE. A good approach
-to compile on any cluster (such as wICE) is to launch an interactive job (with the
-``srun`` command).
-
-Many dependencies you might need are centrally installed. The modules that are
-optimized for wICE are available when the appropriate
+Compared to the SkyLake and CascadeLake CPUs on Genius, the wICE nodes
+feature more recent CPU models such as Intel IceLake, Intel Sapphire Rapids
+and AMD Genoa. While architectural differences between SkyLake and CascadeLake
+CPUs can be neglected, the differences with newer CPU models are more
+substantial. When it comes to GPUs there are also significant differences in
+the capabilities of P100, V100, A100 and H100 GPUs.
+
+When locally installing software yourself, we therefore recommend to have
+separate installations for the different types of CPUs and (if applicable)
+GPUs on which you intend to run the software. This rule applies most strongly
+to performance-critical code which is compiled from source (and less strongly
+to light-weight tools based on precompiled binaries).
```

Comment:
precompiled binaries or interpreted code such as pure Python scripts).

Reply:
(changed as part of 5bfa41d2c2a706327ab763c179759536d3c6d765)